galley/pkg/config/analysis/analyzers/testdata/misannotated.yaml

```diff
@@ -95,3 +95,31 @@ spec:
     spec:
       containers:
       - name: fortio
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: staging
+  annotations:
+    # Sidecar injector annotation does not belong to Namespace
+    sidecar.istio.io/inject: "true"
+spec: {}
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: test-endpoints
+  annotations:
+    # Endpoints should not have any Istio annotations
+    sidecar.istio.io/statsInclusionPrefixes: cluster.outbound
+subsets:
+- addresses:
+  - ip: 10.36.0.4
+    nodeName: kubernetes-node
+    targetRef:
+      kind: Pod
+      name: test-endpoints-755599958f-qkrk8
+  ports:
+  - name: http
+    port: 8080
+    protocol: TCP
```

Comment:
You should be able to leave all this out for the unit tests, it's not relevant and we're not doing schema checking here.